buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```diff
@@ -13,11 +13,11 @@ repositories {
   }
 }
 
-val otelVersion = "1.16.0"
-val otelAlphaVersion = "1.16.0-alpha"
-val otelContribAlphaVersion = "1.16.0-alpha"
-val otelInstrumentationVersion = "1.16.0-SNAPSHOT"
-val otelInstrumentationAlphaVersion = "1.16.0-alpha-SNAPSHOT"
+val otelVersion = "1.17.0-SNAPSHOT"
```

Comment:
Usually we change only `otelInstrumentationVersion` and `otelInstrumentationAlphaVersion` to spanpshots.

Reply:
The upstream depends on the stable SDK version, and so should we